ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -201,20 +201,42 @@ def _parse_data_model_columns(
 
         return columns
 
-    def create_dbt_lineage(self) -> Iterable[AddLineageRequest]:
+    def process_dbt_lineage_and_descriptions(self) -> Iterable[AddLineageRequest]:
         """
         After everything has been processed, add the lineage info
         """
-        logger.info("Processing DBT lineage")
+        logger.info("Processing DBT lineage and Descriptions")
         for data_model_name, data_model in self.data_models.items():
+
+            to_entity: Table = self.metadata.get_by_name(
+                entity=Table, fqn=data_model_name
+            )
+            if to_entity:
+                # Patch table descriptions from DBT
+                if data_model.description:
+                    self.metadata.patch_description(
+                        entity=Table,
+                        entity_id=to_entity.id,
+                        description=data_model.description.__root__,
+                        force=self.source_config.dbtConfigSource.dbtUpdateDescriptions,
+                    )
+
+                # Patch column descriptions from DBT
+                for column in data_model.columns:
+                    if column.description:
+                        self.metadata.patch_column_description(
+                            entity_id=to_entity.id,
+                            column_name=column.name.__root__,
+                            description=column.description.__root__,
+                            force=self.source_config.dbtConfigSource.dbtUpdateDescriptions,
+                        )
+
+            # Create Lineage from DBT
```

Comment:
Should we create a new function called `process_description or does it have to tight to the lineage one?

Reply:
New function will require to loop over the same data again so this was done to avoid that